.deploy/accessibility-app/nginx.conf

```diff
@@ -50,15 +50,21 @@ http {
             access_log off;
             log_not_found off;
         }
-    
+
+        location /nginx_status {
+            stub_status on;
+            allow 10.0.0.0/8;
```

Comment:
Could you add a comment saying where this range comes from (so we have a clue when things break)?

Reply:
I will push a comment, but it's the internal K8 network since it's only meant as a potential K8 healthcheck. Although we don't intend to use at this time.